jest.config.mjs

```diff
@@ -166,6 +166,8 @@ const config = {
   // This option allows use of a custom test runner
   // testRunner: "jest-circus/runner",
 
+  // extensionsToTreatAsEsm: ['.ts'],
+
   // A map from regular expressions to paths to transformers
   transform: {
```

Comment:
[suggestion] Looks like this is no longer required if we have changed the preset to `ts-jest/presets/default-esm`.